langchain/src/document_loaders/web/github.ts

```diff
@@ -32,7 +33,7 @@ export interface GithubRepoLoaderParams {
   recursive?: boolean;
   unknown?: UnknownHandling;
   accessToken?: string;
-  ignoreFiles?: (string | RegExp)[];
+  ignorePaths?: string[]
```

Comment:
This will break anyone currently using this functionality. Let's make a new argument name instead and deprecate this one.